packages/core/src/modules/connections/ConnectionsApi.ts

```diff
@@ -227,6 +230,42 @@ export class ConnectionsApi {
     return connectionRecord
   }
 
+
+  /**
+   * Send a trust ping to an established connection
+   *
+   * @param connectionId the id of the connection for which to accept the response
+   * @param responseRequested do we want a response to our ping
+   * @param withReturnRouting do we want a response at the time of posting
+   * @returns boolean indicating success or failure
+   */
+  public async trustPing(connectionId: string, responseRequested=true, withReturnRouting=false): Promise<boolean> {
```

Comment:
For other apis we usually use objects when there's multiple config parameters. Otherwise you'll just see a call with (false, true). If we make it an object, you can directly see where the options are for.

```suggestion
export interface SendPingOptions {
  responseRequested?: boolean
}

  public async sendPing(connectionId: string, { responseRequested = true }: SendPingOptions): Promise<boolean> {
```

Reply:
Fixed